jina/flow/base.py

```diff
@@ -479,10 +479,9 @@ def add(
         uses: Optional[Union[str, Type['BaseExecutor'], dict]] = 'BaseExecutor',
         uses_after: Optional[Union[str, Type['BaseExecutor'], dict]] = None,
         uses_before: Optional[Union[str, Type['BaseExecutor'], dict]] = None,
-        uses_internal: Optional[
-            Union[str, Type['BaseExecutor'], dict]
-        ] = 'BaseExecutor',
-        volumes: Optional[List[str]] = None,
+        override_with_params: Optional[Dict] = None,
```

Comment:
run `scripts/inject-cli-as-overload.py` to auto generate this

Reply:
awesome thanks

Reply:
Done!, thx